mne/preprocessing/ica.py

```diff
@@ -270,6 +271,22 @@ def __repr__(self):
 
         return '<ICA  |  %s>' % s
 
+    def _check_for_unsupported_ica_channels(self, picks, info):
```

Comment:
Yes






On Thu, Jun 2, 2016 at 9:10 PM -0700, "Alexander Rudiuk" <notifications@github.com> wrote:












In mne/preprocessing/ica.py:
> @@ -270,6 +271,22 @@ def __repr__(self):
>  
>          return '<ICA  |  %s>' % s
>  
> +    def _check_for_unsupported_ica_channels(self, picks, info):



Does this just mean moving it outside the class?



—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub, or mute the thread.